mender-deb-package

```diff
@@ -135,6 +134,9 @@ prepare_recipe() {
     if [ "${debian_recipe}" = "debian-2.1.x" ]; then
       # Skip for 2.1.x, special handling in debian/rules
       true
+    elif [ "${debian_recipe}" = "debian-master" ]; then
```

Comment:
@oleorhagen I just realized that this will fail if we update recipes after 4.1.0 for example. I will amend